utilities/api/get_ssc_deps_private_api.py

```diff
@@ -8,9 +8,8 @@ def get_deployment_id():
 
     r = requests.get('https://semgrep.dev/api/v1/deployments',headers=headers)
     if r.status_code != 200:
-        sys.exit(f'Get failed: {r.text}')
-    data = json.loads(r.text)
```

Comment:
It's not necessary because the revised code (line 12) uses `r.json()` which is JSON-formatted. This is a more Pythonic way to get JSON from a response object.

Reply:
ahh sorry i missed `r.json` in the revised code, thanks!